docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -782,54 +782,54 @@ curl http://localhost:3000/ngsi-ld/v1/entities/urn:ngsi-ld:Sensor:S888 \
-3. NGSIv2 プロトコルを使用してクエリを転送: `GET /v2/entities/urn:ngsi-ld:Sensor:S888`4. レスポンスを NGSIv2 → 内部形式 → NGSI-LD に変換してクライアントへ返却
+1. GeonicDB は `urn:ngsi-ld:Sensor:S888` がローカルに存在しないことを検出します
+2. 登録情報から `http://legacy-system.example.com/v2` を特定します
+3. NGSIv2 プロトコルを使用してクエリを転送します: `GET /v2/entities/urn:ngsi-ld:Sensor:S888`4. レスポンスを NGSIv2 → 内部フォーマット → NGSI-LD に変換し、クライアントに返します
 
 ---
 
-## ユースケースとベストプラクティス### どのAPIを使用すべきか？
+## ユースケースとベストプラクティス### どの API を使用すべきか？
 
-#### NGSIv2を選択すべき場合
+#### NGSIv2 を選択すべき場合
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)